### PR TITLE
#2 Endpoints to retrieve image and video metadata

### PR DESCRIPTION
**Pull request by JonaszJestem**

### CODE REVIEW

File: src/core/common/common.types.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export type Orientation = 'landscape' | 'portrait';
```
Comment (KrzysiekWyka) [inline src/core/common/common.types.ts]: Why `Orientation` cannot be enum?

File: src/core/image/image.service.test.ts [hunk 2]
```diff
@@ -45,12 +55,20 @@ describe('ImageService', () => {
...
+        left: expectedDimensions,
+      } as Region;
       const image = await getTestImage();
 
       const croppedImage = imageService.crop(image, cropOptions);
 
-      const colors = await getColors(await streamToBuffer(croppedImage), 'image/jpeg');
+      const colors = await getColors(
+        await streamToBuffer(croppedImage),
+        'image/jpeg',
```
Comment (KrzysiekWyka) [inline src/core/image/image.service.test.ts]: Can u create const for this value?

File: src/core/common/schemas.ts [hunk 3]
```diff
@@ -11,11 +11,3 @@ export const adapterParamsSchema = {
 };
 
 export type AdapterParams = { adapter: string };
-
-export const fileBodySchema = {
```
Comment (KrzysiekWyka) [inline src/core/common/schemas.ts:15]: If we gonna send post request without file what will do?

**Review by KrzysiekWyka — CHANGES_REQUESTED** [review]

File: src/core/image/image.service.test.ts (hunk 2 above)
Comment (JonaszJestem) [reply src/core/image/image.service.test.ts]: done

File: src/core/common/common.types.ts (hunk 1 above)
Comment (JonaszJestem) [reply src/core/common/common.types.ts]: It can. Fixed

File: src/core/common/schemas.ts (hunk 3 above)
Comment (JonaszJestem) [reply src/core/common/schemas.ts:15]: I've added validation for that

**Review by KrzysiekWyka — APPROVED** [review]

**Pull request merged**